src/new/templates/rust/ui/framework-process/test/framework-process-test/framework-process-test/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hyperprocess-test"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = "1.0"
+bincode = "1.3"
+hyperware_process_lib = { version = "1.0.4" }
+process_macros = "0.1.0"
+hyperware_app_common = { git = "https://github.com/hyperware-ai/hyperprocess-macro" }
+rmp-serde = "1.1"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+thiserror = "1.0"
+url = "2.5.3"
+rand = "0.8"
+wit-bindgen = "0.36.0"
```

Comment:
Get rid of unused deps (I suspect at least bincode, rand, rmp-serde, thiserror, url are unused) & sort remaining deps alphabetically